public/app/features/logs/components/LogDetails.tsx

```diff
@@ -77,19 +76,21 @@ class UnThemedLogDetails extends PureComponent<Props> {
     const linksAvailable = links && links.length > 0;
 
     // If logs with error, we are not showing the level color
-    const levelClassName = cx(!hasError && [style.logsRowLevel, styles.logsRowLevelDetails]);
+    const levelClassName = !hasError
+      ? `${levelStyles.logsRowLevelColor} ${rowStyles.logsRowLevel} ${styles.logsRowLevelDetails}`
+      : '';
```

Comment:
Sure. Thought about this but I thought I liked more having more visual information in the left side than in the right side.

Reply:
Yea I know. I'd visibly either miss the `!` in front of `hasError` or the `''`. In the end I am fine with both, was just a suggestion.

Reply:
Yeah, it was understood from the beginning. I share this view, which is why sometimes I try to avoid testing to `!something" or resort to being explicit `something === false`.